qiskit_experiments/framework/experiment_data.py

```diff
@@ -233,6 +234,8 @@ class ExperimentData:
     _metadata_filename = "metadata.json"
     _max_workers_cap = 10
 
+    _db_url = "https://auth.quantum.ibm.com/api"
```

Comment:
Why does this have to be hardcoded? Can we upstream an interface into qiskit-ibm-runtime that we could query so we don't have to hardcode this?

Reply:
You're right, @gadial pointed out that this can be accessed via `backend.service._account.url`. Updated in bf685c7.